src/main/java/org/prebid/cache/repository/redis/RedisRepositoryImpl.java

```diff
@@ -46,14 +49,22 @@ public RedisRepositoryImpl(final RedisPropertyConfiguration redisConfig,
             return Mono.empty();
         }
 
-        return createReactive().setex(normalizedId, expiry, Json.toJson(wrapper))
-                               .map(payload -> wrapper);
+        try {
+            return createReactive().setex(normalizedId, expiry, Json.toJson(wrapper))
+                    .map(payload -> wrapper);
+        } catch (RedisException e) {
+            return Mono.error(new RepositoryException(e.toString()));
```

Comment:
It is a good idea to pass not only exception message to RepositoryException but original exception as well so that it is not lost if someone wants to log it later. This applies to similar block in `findById`